### ISSUE TITLE
Playing next episode in any way doesn't work as expected.

### ISSUE THREAD

**Issue by Djole137** [iwalton3/plex-mpv-shim#121]
When playing episodes in mpv, when the episode finishes or I try to play the next episode using the keyboard shortcuts or I try to change the episode using the Plex webpage, episodes just mark as finished and play next as soon as they're played until it goes to the end of the series.

I tried using both external and internal mpv, tried deleting all scripts, using different browsers and casting through the apps. Also tried using both app.plex.tv and the local webpage.

I'm on EndeavourOS Linux. The issue has been happening for a few months but it was working fine before that so I assume it's a Plex update that broke the script.

**Comment by phob** [comment]
Yeah I have the same problem. I think this project isn't actively maintained anymore...

**Comment by Djole137** [comment]
@phob  I found this fork which fixes the issue for me. I just installed it using pip. 
https://github.com/elldw/plex-mpv-shim

**Comment by enchained** [comment]
> [@phob](https://github.com/phob) I found this fork which fixes the issue for me. I just installed it using pip. https://github.com/elldw/plex-mpv-shim

Looks like that fork is gone. Do you have any source files of the fix left, something that could be used to create a pull request?

**Comment by Djole137** [comment]
I don't but I found a fork that includes the fix.
https://github.com/Tracreed/plex-mpv-shim
